### ISSUE TITLE
Issue with on-click with no quotations

### ISSUE THREAD

**Issue by richardanaya** [PolymerLabs/lit-html#144]
Works:
```javascript
html`
<div><button on-click=${()=>window.alert("hey")}></button></div>
`
```

Doesn't work:
```javascript
const name = "Richard";
html`
<div>${name}<button on-click=${()=>window.alert("hey")}></button></div>
`
```
https://codepen.io/anon/pen/JrzzNd

It creates invalid HTML:

```html
<div>Richard<button on-click="<!--_-lit-html-_--">&gt;</button></div>
```

**Comment by justinfagnani** [comment]
I think this is fixed by: https://github.com/PolymerLabs/lit-html/pull/134

**Comment by justinfagnani** [comment]
This was fixed in #134
